src/models/user.js

```diff
@@ -1,34 +1,27 @@
-/* eslint-disable linebreak-style */
 const mongoose = require('mongoose');
 
-UserSchema = new mongoose.Schema(
+const { Schema } = mongoose;
 
-{
-  Name:{
+const UserSchema = new Schema({
+  email: {
     type: String,
-    required: true
+    required: true,
+    trim: true
   },
-  email:{
+  password: {
     type: String,
     required: true
   },
-  city:{
-    type: String,
-    required: true
-  },
-  phone:{
-    type: String,
-    required:true
-  },
-  bloodgroup:{
+  role: {
     type: String,
-    required: true
+    default: 'basic',
+    enum: ['donor', 'hosipital', 'admin']
   },
-  address:{
-    type: String,
-    required: true
+  accessToken: {
```

Comment:
I don't think we need to save users accessToken in users table/collection. Thought we can have a seperated table to store expired tokens or if user logged off. 
Well done bro